esp-hal/src/lib.rs

```diff
@@ -70,6 +70,49 @@
 // MUST be the first module
 mod fmt;
 
+#[cfg(esp32)]
```

Comment:
Should we move this to the soc module for each chip?

Reply:
That's a good idea! 💡